src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala

```diff
@@ -33,30 +35,38 @@ object ClusterMonitorSupervisor {
   case class ClusterStarted(cluster: Cluster) extends ClusterSupervisorMessage
   // sent after a cluster is updated by the user
   case class ClusterUpdated(cluster: Cluster) extends ClusterSupervisorMessage
-
   // sent after cluster creation fails, and the cluster should be recreated
   case class RecreateCluster(cluster: Cluster) extends ClusterSupervisorMessage
   // sent after cluster creation succeeds, and the cluster should be stopped
   case class StopClusterAfterCreation(cluster: Cluster) extends ClusterSupervisorMessage
+  //Sent when the cluster should be removed from the monitored cluster list
+  case class RemoveFromList(cluster: Cluster) extends ClusterSupervisorMessage
   // Auto freeze idle clusters
   case object AutoFreezeClusters extends ClusterSupervisorMessage
+  // Timers ADT
```

Comment:
I'm not sure this ADT makes sense. As I understand it, `TimerKey` is just a key to reference the timer instance (not a message) and `Tick` is the message to send. So how about:
```
case object TimerKey
case object CheckForClusters extends ClusterSupervisorMessage
```